tests/e2e/fixtures/storage_fsx_dependencies.py

```diff
@@ -236,16 +111,25 @@ def on_create():
         fsx_pvc["spec"]["volumeName"] = claim_name
         write_cfg(fsx_pvc, fsx_pvc_filepath)
 
-        kubectl_apply(fsx_sc_filepath)
         kubectl_apply(fsx_pv_filepath)
         kubectl_apply(fsx_pvc_filepath)
 
         fsx_claim["claim_name"] = claim_name
+        fsx_claim["file_system_id"] = file_system_id
 
     def on_delete():
         kubectl_delete(fsx_pvc_filepath)
         kubectl_delete(fsx_pv_filepath)
 
+        details_fsx_volume = metadata.get("fsx_claim") or fsx_claim
+        # file_system_id = details_fsx_volume["file_system_id"]
```

Comment:
Left over comment block?

Reply:
no actually, has commented for some testing. Uncommented it.